### PR TITLE
USHIFT-4026: Add tests for MicroShift published image validation

### PR DESCRIPTION
**Pull request by ggiguash**

### CODE REVIEW

**Comment by ggiguash** [comment]
The 4.18-ec3 MicroShift package from mirror.openshift.com is one month old and some tests are not passing. I will unhold this PR when the packages are rebuilt.

File: test/kickstart-templates/includes/main-ostreecontainer.cfg [hunk 1]
```diff
@@ -1,22 +1,33 @@
...
+# image names. These need to be redirected to the local mirror location.
 %pre-install --log=/dev/console --erroronfail
 
 mkdir -p /etc/containers/registries.conf.d
-cat > /etc/containers/registries.conf.d/998-microshift-bootc-registry.conf <<EOF
+# Enable insecure registry access from the hypervisor to allow local image pull.
+# See https://github.com/containers/bootc/blob/main/docs/src/registries-and-offline.md#insecure-registries
+cat > /etc/containers/registries.conf.d/998-microshift-bootc-registry.conf <<'EOF'
 [[registry]]
+    prefix = "docker.io/library"
```
Comment (ggiguash) [inline test/kickstart-templates/includes/main-ostreecontainer.cfg:11]: @pmtk , I created the [USHIFT-5083](https://issues.redhat.com/browse/USHIFT-5083) follow-up as discussed to make sure we always use qualified image paths.

Comment (pmtk) [reply test/kickstart-templates/includes/main-ostreecontainer.cfg:11]: Thanks! I totally forgot about that one

**Pull request merged**